e2e/rdctl.e2e.spec.ts

```diff
@@ -716,7 +716,9 @@ test.describe('Command server', () => {
                   });
                   const settings = JSON.parse(stdout);
 
-                  expect(['version', 'kubernetes', 'portForwarding', 'images', 'telemetry', 'updater', 'debug', 'pathManagementStrategy', 'diagnostics']).toMatchObject(Object.keys(settings));
+                  expect(['version', 'kubernetes', 'portForwarding', 'images', 'telemetry',
+                    'updater', 'debug', 'pathManagementStrategy', 'diagnostics'])
+                    .toEqual(expect.arrayContaining(Object.keys(settings)));
```

Comment:
`arrayContaining` doesn't check that we don't have extras.  Consider instead:

```ts
const expected = new Set(['version', 'kubernetes', 'portForwarding', 'images', 'telemetry',
  'updater', 'debug', 'pathManagementStrategy', 'diagnostics']);

expect(expected).toEqual(new Set(Object.keys(settings)));
```

Reply:
Done, and refactored into a method because two other tests were making the same kind of test.